bundle/manifests/remediation.medik8s.io_nodehealthchecks.yaml

```diff
@@ -20,10 +22,14 @@ spec:
         description: NodeHealthCheck is the Schema for the nodehealthchecks API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
I'm not sure if you can have line breaks in the value of a field

Reply:
this is what the latest operator-sdk did(I upgraded). I'll recheck